apps/communications/dialer/js/dialer.js

```diff
@@ -36,9 +36,13 @@ var NavbarManager = {
 var CallHandler = (function callHandler() {
   var COMMS_APP_ORIGIN = document.location.protocol + '//' +
     document.location.host;
+  var callScreenWindow = null;
```

Comment:
Maybe we should keep the callscreen management into a separate file to avoid the 500+ LOC file.